### ISSUE TITLE
ingest failure - list() function in config file taken as string

### ISSUE THREAD

**Issue by artturi** [opencast/pyCA#22]
I'm trying to get pyca working so this may not be a pyca issue but an error from my side.

Here's an extract from the config file : 
    [capture]
    directory        = './recordings'
    command           = 'timeout {{time}} gst-launch v4l2src device=/dev/video0 ! video/x-raw-rgb ! ffmpegcolorspace ! xvidenc ! avimux ! filesink location={{dir}}/{{name}}.avi | :'
    flavors          = list('presenter/source')
    files            = list('{{dir}}/{{name}}.avi')

And here's the log output from pyca when the recording occurs : 

> 2015-06-01 10:45:01 INFO     [ca.py:318:ingest()] Adding track (l)
> 2015-06-01 10:45:01 ERROR    [ca.py:233:start_capture()] Something went wrong during the upload
> 2015-06-01 10:45:01 ERROR    [ca.py:234:start_capture()] Traceback (most recent call last):
>  File "pyca/ca.py", line 231, in start_capture
>    workflow_config)
>  File "pyca/ca.py", line 322, in ingest
>    mediapackage = http_request('/ingest/addTrack', fields)
>  File "pyca/ca.py", line 274, in http_request
>    curl.perform()
> error: (26, 'couldn\'t open file "l"\n')

Any idea ?

**Comment by artturi** [comment]
I forgot to give some precisions :
- os is ubuntu 12.04
- python's version is 2.7.3
- installation was done with virtualenv, following the wiki's instructions
- I start pyca in virtualenv (I guess that from the "(venv)" prefix in the prompt)

**Comment by lkiesow** [comment]
I have just tried it on Ubuntu 12.04 without any problems. Can you try the following:
- Test pyCA with default settings (FFmpeg test source), only set your server
- Test pyCA against `mhtest.virtuos.uos.de` or `testallinone.usask.ca`
- If the latter does not work, can you send me the full configuration file

**Comment by artturi** [comment]
Quick update :
- test with default settings works
- test against mhtest.virtuos.uos.de fails with same error

**Comment by lkiesow** [comment]
I may have found the reason for it to fail.
Can you try setting 

```
flavors = 'presenter/source'
```

instead of 

```
flavors = list('presenter/source')  <-  This will raise an exception
```

It seems like the `list(…)` syntax is only used for setting the defaults

**Comment by lkiesow** [comment]
Solved the problem.
Documentation is fixed with commit  8cd3e1d
